AK/Time.cpp

```diff
@@ -234,9 +234,9 @@ Time Time::operator-(Time const& other) const
     VERIFY(other.m_nanoseconds < 1'000'000'000);
 
     if (other.m_nanoseconds)
-        return *this + Time((i64) ~(u64)other.m_seconds, 1'000'000'000 - other.m_nanoseconds);
+        return *this + Time(static_cast<i64>(~static_cast<u64>(other.m_seconds)), 1'000'000'000 - other.m_nanoseconds);
 
-    if (other.m_seconds != (i64)-0x8000'0000'0000'0000)
+    if (other.m_seconds != static_cast<i64>(-0x8000'0000'0000'0000))
```

Comment:
That literal schould already be an i64

Reply:
For some reason the compiler thinks that that's unsigned.

Reply:
Ahh the 8 at the beginning sets the sign bit
So it's the literal that would represent i64Min, which is larger than i64Max, by one, so why are we taking the negative here then, don't we want just i64Max then?

Reply:
Or it is the integer indefinite, but that should only be 0x8.... And not with an extra -, but
¯\_(ツ)_/¯

Reply:
AK::Time intentionally doesn't include limits or stdint or anything because of how it's used. Not sure if that's valid for the cpp file but there's a comment in the header